sqlalchemy_jsonapi/serializer.py

```diff
@@ -1079,6 +1081,7 @@ def post_collection(self, session, data, api_type):
             session.rollback()
             raise ValidationError(str(e.orig))
         except AssertionError as e:
+            # pragma: no cover
```

Comment:
There is no coverage here because I cannot try and raiser ValidationError(e.msg) because no msg attribute is on the AssertionError.